skimage/util/_backends.py

```diff
@@ -0,0 +1,137 @@
+import functools
+from importlib.metadata import entry_points
+from functools import cache
+import os
+import warnings
+
+
+def dispatching_disabled():
+    """Determine if dispatching has been disabled by the user"""
+    no_dispatching = os.environ.get("SKIMAGE_NO_DISPATCHING", False)
+    if no_dispatching == "1":
+        return True
+    else:
+        return False
+
+
+def public_api_name(func):
```

Comment:
Just a nitpick, from the function name I'd expect this to return the module name + the function name. But since the function only returns the module name, how about we name this something like `public_api_module` or similar. 

(I feel like I discussed this before and might have even suggested the old `public_api_name` :smile:)